ApexOSA/powermeter.py

```
@@ -0,0 +1,85 @@
+        """
+        return "Powermeter of " + str(self.__ID)
+
+    def SetUnit(self, Unit):
+        """
+        Set the power unit of the Powermeter equipment
+        Unit is a string which could be "dBm" for logaritmic or "mW" for linear power
+        """
+
+        try:
+            Unit = str(Unit)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)